File: java/MinimalGL/app/src/main/java/org/example/minimalgl/MainActivity.java
package org.example.minimalgl;

import android.support.v7.app.ActionBarActivity;
import android.os.Bundle;

public class MainActivity extends ActionBarActivity {

	@Override protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		_view = new GLSurfaceViewImpl(getApplication());
		setContentView(_view);
	}

	private GLSurfaceViewImpl _view;
}

File: kotlin/DatePicker/app/src/main/java/com/example/datepicker/MainActivity.kt
package com.example.datepicker

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
import java.util.*

class MainActivity : AppCompatActivity() {

	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		setContentView(R.layout.activity_main)

		val today = Calendar.getInstance()
		datePicker.init(today.get(Calendar.YEAR), today.get(Calendar.MONTH), today.get(Calendar.DAY_OF_MONTH)) {
			view, year, monthOfYear, dayOfMonth ->
				val month = monthOfYear + 1
				val msg = "Selected Date is $dayOfMonth/$month/$year"
				Toast.makeText(this@MainActivity, msg, Toast.LENGTH_SHORT).show()

				textView.text = msg
		}

	}
}
